### DISCUSSION TITLE
Adding support for external blacklists

### DISCUSSION THREAD

**Discussion by AnderssonPeter** [tomMoulard/fail2ban#65] [Ideas]
Hi any chance you could add support for external black lists.
Example you give the plugin a list of urls, and set a refresh timer.

Example to block shodan you could give it: https://raw.githubusercontent.com/SilvrrGIT/IP-Lists/master/shodan or to block tor you could give it https://check.torproject.org/exit-addresses (Has a slightly different syntax)

**Comment by tomMoulard** [comment]
Kind of relates to #39
